AirSkyBoat/AirSkyBoat
# Fix familiar and intimidate issues

**_I affirm:_**


## Please enter a player-facing description
- The same mob may now be familiared by multiple players as long as the mob uncharms between each familiar. (Tracent)
- The intimidation from Gration now retail-accurately only applies to spells targeting Gration (and thus will, for example, not apply to players curing own alliance members unless some other type of intimidation (e.g., Dragon Killer) is at play). (Tracent)
- Ying and Yang are now immune to gravity as in era. (Tracent, Siknoz)

## What does this pull request do? (Please be technical)
- The familiar change fixes an issue whereby after a player familiars a charmed mob and then later leaves the mob, another player stumbles upon the mob and is unable to familiar because it has already been familiared once. The fix essential removes the familiar HP boost and local familiar variable on uncharm. Also changes the familiar logic to use standard calls and mods rather than custom logic.
- The intimidation change makes Gration use humanoid killer mod rather than intimidation status effect thus fixing the issue described in player facing description. For behavior era video of Gration [here](https://www.youtube.com/watch?v=j-raD3hu3rY), also many others on youtube.
- Ying and Yang should be immune to gravity as per Siknoz Capture.

## Steps to test these changes
- Fight Gration (to get on hate list) and then try casting on gration versus casting on self, alliance, and other nearby mobs.
- Charm a mob and check max HP then familiar and make sure HP has increased by 10%, then leave the mob, then check max HP again to see the 10% boost is gone, then charm and familiar again and check max HP.

Closes https://github.com/HorizonFFXI/HorizonXI-Issues/issues/1340

## Special Deployment Considerations
